web-client/src/main/java/no/nav/familie/webflux/builder/ProxyTimeout.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.webflux.builder
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class ProxyTimeout(
+    @Value("\${familie.proxy.timeout.connect:2000}") val connectTimeout: Long,
```

Comment:
Som eg forstår det var denne 15 sekund før, men blir no kutta til to. Har ikkje grunnlag for å seie om det blir problem eller ikkje. I så fall kan vi jo sjølvsagt endre det anten her eller via property, men.

Reply:
Min erfaring for timeout generellt er at det virker på første eller andre forsøket, dvs TCP prøver på nytt ved 1 eller 2 sekunder. 
Hvis det ikke virker, så prøver den på nytt men kommer aldri å lykkes vid de andre forsøkene. Så en høy connection timeout vil aldri gi noe

Reply:
Ja, sant nok, eg deler den generelle erfaringa. Så vil tru at dette er greitt.